group/assignment5/test1.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <string>
+#include "WordDependencyGraph.h"
+#include "test_library.h"
+
+using namespace std;
+
+
+int main() {
+	// test 1
```

Comment:
I believe more interesting test cases could be added here: empty dictionary, empty words in the dictionary, identical words, etc. In addition it's sometimes useful to make tests separate and give them clear names so that if test fails you (or not you) can see what exactly went wrong :)